pyproject.toml

```diff
@@ -1,6 +1,57 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
-]
+requires = ["setuptools~=66.1.1", "wheel~=0.38.4"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "luxtronik"
+version = "0.0.0-dev"
+license = {text = "MIT"}
+readme = "README.md"
+authors = [
+  {name = "Bouni", email = "bouni-luxtronik@owee.de"}
+]
+description = "A luxtronik heatpump controller interface"
+keywords    = ["heatpump", "luxtronik2"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Heatpump interface",
+]
+requires-python = ">=3.9.0"
+
+[project.urls]
+"Source Code" = "https://github.com/bouni/python-luxtronik/"
+"Bug Reports" = "https://github.com/bouni/python-luxtronik/issues/"
+"Docs: Dev"   = "https://github.com/bouni/python-luxtronik/"
+
+[project.scripts]
+luxtronik = "luxtronik.__main__:main"
+
+[project.optional-dependencies]
+dev = [
+  "pytest~=7.2.1",
+  "autoflake~=2.0.0",
+  "black~=22.12.0",
+  "flake8-comprehensions~=3.10.1",
+  "flake8-docstrings~=1.6.0",
+  "flake8-noqa~=1.3.0",
+  "flake8~=6.0.0",
+  "isort~=5.11.4",
+  "pycodestyle~=2.10.0",
+  "pydocstyle~=6.2.3",
+  "pyflakes~=3.0.1",
+  "pyupgrade~=3.3.1",
+  "ruff~=0.0.231",
+  "yamllint~=1.28.0"
```

Comment:
Do we need the version pinning here? How are we going to manage this? How often are we supposed to bump those versions?

Reply:
Those are shamelessly stolen out of HA's pyproject.toml 😇 
But I guess you're right, better not pin them.

Reply:
Well, version pinning is a hot topic and you could argue back and forth all time long. I just wonder how we're supposed to manage those numbers. I don't know all of those libraries/modules/tools, so I have no idea, which version to use. Sticking with old version for forever is probably also not a good idea.

We could add a reference to the Home Assistant file and check/compare it every now and then. I'm sure someone in the Home Assistant project takes care about such things :-).